src/components/QuicksightDashboardPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Store } from "redux";
+import { RootState } from "../store";
+import * as PropTypes from "prop-types";
+import Header from "./Header";
+import Footer from "./Footer";
+import title from "../utils/title";
+import QuicksightDashboard from "./QuicksightDashboard";
+
+export interface QuicksightDashboardPageProps
+  extends React.Props<QuicksightDashboardPageProps> {
+  params: {
+    library?: string;
+  };
+}
+
+export interface QuicksightDashboardPageContext {
+  editorStore: Store<RootState>;
+}
+
+/** Page holds quicksight dashboards. */
+export default class QuicksightDashboardPage extends React.Component<
+  QuicksightDashboardPageProps
+> {
```

Comment:
Since we no longer need `QuicksightDashboardPageProps`, we can change the props type here to none, like this:
```typescript
export default class QuicksightDashboardPage extends React.Component<never> {
...
```